### DISCUSSION TITLE
✘ [ERROR] Big integer literals are not available in the configured target environment ("chrome87", "edge88", "es2019", "firefox78", "safari13.1")

### DISCUSSION THREAD

**Discussion by kydner** [quasarframework/quasar#16071] [General - Components / Directives / etc]
I found this error when i use library v-money3, How fix it?

**Comment by suleimanas** [comment]
i believe you need **es2020** to use BigInt

**Reply by kydner** [reply]
i use es2019, but i set to 2020 still error
target: {
        browser: ['es2020', 'edge88', 'firefox78', 'chrome87', 'safari13.1'],
        node: 'node16'
 },

**Reply by suleimanas** [reply]
try to set vite config to: 
`optimizeDeps.esbuildOptions.target: 'es2020'` 
and 
`build.target: 'es2020'`

**Reply by kydner** [reply]
build: {
  optimizeDeps: {
    esbuildOptions: {
      target:  'es2020'
    }
  },
  target: 'es2020'
}
like this on quasar.config.js ?

**Reply by suleimanas** [reply]
no inside build > extendViteConf like this:

```
build: {
...

  extendViteConf(config) {
    config.optimizeDeps = { ... }
  }
}
```

**Reply by kydner** [reply]
```
extendViteConf(conf) {
conf.optimizeDeps = {
esbuildOptions: {
target: 'es2020'
}
}
},
```
i has change, no error but page is blank
